test/parallel/test-util-inspect.js

```diff
@@ -434,7 +434,7 @@ assert.strictEqual(
 
 // Array with extra properties.
 {
-  const arr = [1, 2, 3, , ];
+  const arr = [1, 2, 3, , ]; // eslint-disable-line no-sparse-arrays
```

Comment:
most of these comments are using `eslint-disable-next-line`. Should we try to make it consistent and always use `eslint-disable-next-line`?